pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java

```diff
@@ -83,8 +85,8 @@
 import software.amazon.awssdk.utils.AttributeMap;
 
 
-@LocalstackDockerProperties(services = {"kinesis"})
-@Test(enabled = false)
+@LocalstackDockerProperties(services = {ServiceName.KINESIS}, portEdge = "4591", environmentVariableProvider = RealtimeKinesisIntegrationTest.HostnameEnvProvider.class, imageTag = "0.13.0.10")
```

Comment:
can this port be dynamic?

Reply:
Also need fix linter, line too long

Reply:
Hi, @xiangfu0 don't review this PR yet. it is still WIP. this kinesis issue is not reproducible in my local and any other VMs I created. So I raised this PR just to test out what is going wrong.
Will fix linting once I have a working solution.